examples/custom-behavior/select-elements-by-bpmn-kind/index.html

```diff
@@ -116,9 +116,11 @@ <h2>Select elements by BPMN kind</h2>
                             <div class="radio">
                                 <label><input type="radio" name="poolSelectionMethod" value="hide" checked> Hide others</label>
                             </div>
+                            <!-- Disable collapsing: see https://github.com/process-analytics/bpmn-visualization-examples/pull/306
```

Comment:
Is it an omission ?
If not, no need a single radio button for hide 🙂

Reply:
I wanted to limit the modifications especially in the js code.
Here is what we see for now
![image](https://user-images.githubusercontent.com/27200110/159034455-004ff01c-b6fa-4c2e-b504-e93b92a4477a.png)

Reply:
I found weird, but ok 👍🏻

Reply:
@csouchet based on your feedback, I removed the radio button in 406eb9f

![image](https://user-images.githubusercontent.com/27200110/159211740-7ac91a38-d228-4837-821b-611153b1759c.png)